tests/testthat/test-create_bcgov_project.R

```diff
@@ -106,6 +106,27 @@ test_that("create_bcgov_project works with empty dir_struct", {
   expect_true(file.exists(file.path(dir, paste0(basename(dir), ".Rproj"))))
 })
 
+test_that("create_bcgov_project works with shiny app dir_struct", {
+  setwd("~")
+  dir <- unique_temp_dir(pattern = "create_foo")
+  ret <- capture.output(create_bcgov_project(dir, coc_email = "me@gov.bc.ca", 
+                                             dir_struct = c("app/", "app.R"),
```

Comment:
I don't think it matters for this issue but I would have thought the `dir_struct` would be `c("app/", "app/app.R")`

Reply:
Changed this.